### ISSUE TITLE
Set study email at study rather than lab level

### ISSUE THREAD

**Issue by becky-gilbert** [lookit/lookit-api#1351]
This is one part of this larger issue about improving researcher/participant communication: https://github.com/lookit/lookit-api/issues/1314

# TL;DR

Set the study's "from"/"reply to" email address at the level of the study rather than lab, so that emails sent to participants by the researcher (via the CHS messaging system) are sent from this study-specific email address instead.

# Narrative

This will make it easier for participant's replies to get to the correct researcher(s), especially for multi-site/lab projects/studies.

# Implementation notes

We'll need to add a new field to the study model. We currently have a 'researcher contact information' that requires the researcher to enter an email address, so we can separate out the email address into a different field and use that (1) when sending emails about the study, and (2) along with the 'researcher contact' text field to construct the existing text in that field ("<researcher name> (contact: <study email>)". We can change the researcher contact field to just require the researcher's name.

There may be a problem with the way that we're doing this now, which is sending emails from the mit.edu domain with a different 'from' address (see: https://github.com/lookit/lookit-api/issues/1340). But in any case, we should be able to find a way to do this that meets the updated requirements being implemented by email providers. We may need to do something like use the MIT domain as the 'from' address, but send the message 'on behalf of <study email>' and include the study email as the reply-to address (if we we're allowed to use a different domain there).
